samples/python-flask/13.core-bot/main.py

```diff
@@ -29,6 +29,19 @@
 SETTINGS = BotFrameworkAdapterSettings(app.config['APP_ID'], app.config['APP_PASSWORD'])
 ADAPTER = BotFrameworkAdapter(SETTINGS)
 
+# Catch-all for errors.
+async def on_error(context: TurnContext, error: Exception):
+    # This check writes out errors to console log
+    # NOTE: In production environment, you should consider logging this to Azure
+    #       application insights.
+    print(f'\n [on_turn_error]: { error }')
```

Comment:
>print(f'\n [on_turn_error]: { error }') [](start = 4, length = 39)

FYI: I've seen where prints get swallowed (can't recall on linux/windows on flask/django).  I've been getting in the practice of using:
print(f"blah-blah", file=sys.stderr)
Which appears to print slightly more consistently.

Reply:
solved